lib/open_api_spex/plug/swagger_ui.ex

```diff
@@ -5,6 +5,14 @@ defmodule OpenApiSpex.Plug.SwaggerUI do
   The full path to the API spec must be given as a plug option.
   The API spec should be served at the given path, see `OpenApiSpex.Plug.RenderSpec`
 
+  ## Configuring SwaggerUI version
+
+  Latest version of SwaggerUI is used by default, but it is possible to configure a different version:
+
+  ```elixir
+  config :open_api_spex, :swagger_ui_version, "4.14.0"
```

Comment:
I realised it would be more convenient and easy to [document](https://hexdocs.pm/open_api_spex/OpenApiSpex.Plug.SwaggerUI.html#init/1-options) as a a plug option:

```elixir
 get "/docs/api", OpenApiSpex.Plug.SwaggerUI, 
  path: "/my_service/docs/api_spec", 
  swagger_ui_version: "4.14.0"
```

The @html is already an EEx template, so it's quite straightforward to make it a template variable.

Reply:
I was thinking that too at one point, but then realized that currently all the plug options are passed to `SwaggerUIBundle` constructor and having one configuration parameter, which is handled differently seemed not correct thus your original proposal to configure it via config seemed more reasonable. Especially since there's already other configuration values, which are configued via config for `open_api_spex`.

After this feedback, do you still think that it would make more sense to introduce it as a plug configuration instead?